cre/capabilities/blockchain/evm/v1alpha/client.proto

```diff
@@ -254,7 +255,7 @@ service Client {
   rpc RegisterLogTracking(RegisterLogTrackingRequest) returns (google.protobuf.Empty);
   rpc UnregisterLogTracking(UnregisterLogTrackingRequest) returns (google.protobuf.Empty);
   rpc LogTrigger(FilterLogTriggerRequest) returns (stream Log);
-  rpc WriteReport(WriteReportRequest) returns (WriteReportReply);
+  rpc WriteReport(WriteReportRequest) returns (sdk.v1alpha.ReportResponse);
```

Comment:
Why this change?. WriteReportRequest should be updated to use ReportResponse as input instead of SignedReport right?

Reply:
ReportResponse is a signed report. This is the type the SDK returns. There's no reason that the workflow author should need to massage protos to write it.